### PR TITLE
#65 Migrating from legacy to container-based infrastructure

### PR DESCRIPTION
**Pull request by tomislater**
#65 Migrating from legacy to container-based infrastructure + allowing failures on jobs running on OS X

### CODE REVIEW

**Comment by tomislater** [comment]
> coverage/coveralls — Coverage decreased (-100.0%) to 0.0%

I don't get it :]

**Comment by fizyk** [comment]
This could be either related to containers, since other project I'm working on also gets 0% coverage. If I remember new projects are already on containers, or it might be related to coveralls migration: http://blog.coveralls.io/2015/07/23/Downtime-From-Big-Migration/

File: .travis.yml [hunk 1]
```diff
@@ -26,6 +26,10 @@ script:
     - "if [[ $TRAVIS_PYTHON_VERSION != '3.2' ]]; then for f in *.rst; do rst-lint $f; done; fi"
 after_success:
     - coveralls
+matrix:
+    allow_failures:
+        - os: osx
```
Comment (fizyk) [inline .travis.yml]: isn't that for ignoring errors? from osx? If we're not running on osx, why would we ignore errors? ;)

Comment (tomislater) [inline .travis.yml]: > isn't that for ignoring errors? from osx?

Yes, we should remove this if tests on OS X will start work.

> If we're not running on osx, why would we ignore errors? ;)

But, we want, right? :> Currently, we know that some errors are occurred. You have seen this issue #54

**Comment by tomislater** [comment]
TODO:
* fix tests on OS X
* fox coverage (container)

**Comment by tomislater** [comment]
It seems to work.

**Comment by fizyk** [comment]
:+1:

**Pull request merged**